<ui version="4.0" >
 <class>QTwikeBMSClass</class>
 <widget class="QWidget" name="QTwikeBMSClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QTwikeBMS</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>BMS State</string>
   </property>
   <widget class="QRadioButton" name="radio_on" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>On</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_off" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Off</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="textField" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>400</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Misc Fixed</family>
    </font>
   </property>
   <property name="undoRedoEnabled" >
    <bool>false</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="centerOnScroll" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>20</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Main</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Charge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>20</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>50</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>80</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ah</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinVoltage" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum" >
     <double>999.899999999999977</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinCurrent" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>50</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum" >
     <double>-25.000000000000000</double>
    </property>
    <property name="maximum" >
     <double>25.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinCharge" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>80</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum" >
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum" >
     <double>30.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
